greenlight/tests/functional/keyboard_shortcuts/test_browser_window.py

```diff
@@ -39,9 +39,11 @@ def test_search_field(self):
 
         keys = [self.keys.ACCEL]
         if self.marionette.session_capabilities['platformName'] == 'LINUX':
-            keys.append(self.browser.get_localized_entity('searchFocusUnix.commandkey'))
+            keys.append(self.browser.get_localized_entity(
+                        'searchFocusUnix.commandkey'))
```

Comment:
There's no strict line length limit for Gaia UI. I personally try to stick to 80 characters unless going over makes the code considerably more readable.